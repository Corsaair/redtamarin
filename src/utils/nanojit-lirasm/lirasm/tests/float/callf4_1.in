; This Source Code Form is subject to the terms of the Mozilla Public
; License, v. 2.0. If a copy of the MPL was not distributed with this
; file, You can obtain one at http://mozilla.org/MPL/2.0/.

a = immf4 1 2 4 9
res = callf4 callf4_sqrt cdecl a
retf4 res
