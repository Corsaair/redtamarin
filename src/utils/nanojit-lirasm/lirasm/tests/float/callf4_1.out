Output is: 1,1.41421,2,3
